lib/libspl/include/sys/types.h

```diff
@@ -74,4 +74,13 @@ typedef union {
 
 #include <sys/param.h> /* for NBBY */
 
+#ifndef _WIN32
```

Comment:
This should be in lib/libspl/include/umem.h. Also, the comment should precede this and there are some check style failures that need to be addressed.